lib/screens/UsernameAddSheet.dart

```diff
@@ -0,0 +1,74 @@
+import 'package:flutter/cupertino.dart' as c;
+import 'package:flutter/widgets.dart';
+import '../styles/style.dart';
+
+class UsernameAddSheet extends StatefulWidget {
+  @override
+  _UsernameAddSheetState createState() => _UsernameAddSheetState();
+}
+
+class _UsernameAddSheetState extends State<UsernameAddSheet> {
+  TextEditingController _username;
+
+  @override
+  void initState() {
+    super.initState();
+    _username = new TextEditingController();
+  }
+
+  @override
+  void dispose() {
+    super.dispose();
+    _username.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    double h=c.MediaQuery.of(context).size.height;
+    return c.CupertinoPageScaffold(
+      child: c.SingleChildScrollView(
+        child: c.Padding(
+          padding: const EdgeInsets.all(15.0),
+          child: c.Column(
+            children: [
+              c.SizedBox(height:h*0.15),
+              c.Align(
+                alignment: c.Alignment.topLeft,
+                child: c.Text('Hello there! ',
+                    style: Style.veryLargeTitle(context)
+                ),
+              ),
+              c.SizedBox(height:10.0),
+              c.Text("So nice to meet you! What do your friends call you?",
+                  style: Style.subTitle(context)
+              ),
+              c.SizedBox(height:h*0.13),
+              c.CupertinoTextField(
```

Comment:
For dark mode, you need to make corrections to make it visible.
<img width="326" alt="Screenshot 2021-01-07 at 12 25 03 AM" src="https://user-images.githubusercontent.com/35165481/103811496-cbc94a00-507e-11eb-9656-a5bb3313e21a.png">

You will find ios design resource here https://developer.apple.com/design/resources/